### PR TITLE
Fix resources link

### PR DESCRIPTION
**Pull request by get-itips**
The original link was a fasttrack resource link, it was then changed to a link that does not have the resources mentioned, so replacing it with a link to several template resources

closes https://github.com/MicrosoftDocs/OfficeDocs-SharePoint/issues/3051

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: migration/Fileshare-to-ODSP-migration-guide.md [hunk 1]
```diff
@@ -176,9 +176,9 @@ Develop a plan for sending communications to your user base, providing clear sta
 ### Onboarding related resources
 
 - [Microsoft 365 end user adoption guide](/office365/customlearning/champ_o365guide): Outlining methodology and resources for implementing proven adoption success factors
-- [Posters, email templates](https://support.microsoft.com/office/create-and-use-site-templates-in-sharepoint-server-versions-60371b0f-00e0-4c49-a844-34759ebdd989): Customizable templates to generate internal awareness and excitement.
+- [Posters, email templates](https://www.microsoft.com/fasttrack/resources): Customizable templates to generate internal awareness and excitement.
```
Comment (JohanFreelancer9) [inline migration/Fileshare-to-ODSP-migration-guide.md:179]:
```suggestion
- [Posters, email templates](https://www.microsoft.com/fasttrack/resources): Customizable templates to generate internal awareness and excitement
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@JoanneHendrickson Thank you for submitting feedback and contributing to the docs. We are currently investigating this.

**Comment by get-itips** [comment]
@scanum can you help me get this merged? thanks

**Comment by get-itips** [comment]
@tiburd tina can you help me with the merging?

**Pull request merged**